### PR TITLE
Renaming default.env -> .env

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Bug fix -- when I run locally using the file named default.env, docker-compose doesn't detect the environment variables. Fixing that in this PR.

### CODE REVIEW

**Review by doublebyte1 — CHANGES_REQUESTED** [review]
@KilimAnnejaro Well spotted! 👍🏽 

It seems that docker compose ignores environment variables defined in the `env_file`. Since there is no need to specify `.env`, could you remove these lines?

https://github.com/geopython/pygeoapi-examples/blob/656579e1c1179d00ba4d2eb8156a989282fbf96c/docker/elastic/docker-compose.yml#L37-L38

Just make sure to mention somewhere in the README that you are using an `.env` file Thank You! 🙏🏽

File: docker/elastic/docker-compose.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ services:
 
   pygeoapi:
     env_file:
-      - default.env
+      - .env
```
[the remark this answers is not in the document]
Comment (KilimAnnejaro) [reply docker/elastic/docker-compose.yml:38]: Tested removing the env_file locally, with this result:

```
pygeoapi_es  | START /entrypoint.sh
pygeoapi_es  | Trying to generate openapi.yml
pygeoapi_es  | Traceback (most recent call last):
pygeoapi_es  |   File "/usr/local/bin/pygeoapi", line 33, in <module>
pygeoapi_es  |     sys.exit(load_entry_point('pygeoapi', 'console_scripts', 'pygeoapi')())
pygeoapi_es  |   File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1157, in __call__
pygeoapi_es  |     return self.main(*args, **kwargs)
pygeoapi_es  |   File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1078, in main
pygeoapi_es  |     rv = self.invoke(ctx)
pygeoapi_es  |   File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1688, in invoke
pygeoapi_es  |     return _process_result(sub_ctx.command.invoke(sub_ctx))
pygeoapi_es  |   File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1688, in invoke
pygeoapi_es  |     return _process_result(sub_ctx.command.invoke(sub_ctx))
pygeoapi_es  |   File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1434, in invoke
pygeoapi_es  |     return ctx.invoke(self.callback, **ctx.params)
pygeoapi_es  |   File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 783, in invoke
pygeoapi_es  |     return __callback(*args, **kwargs)
pygeoapi_es  |   File "/usr/local/lib/python3.10/dist-packages/click/decorators.py", line 33, in new_func
pygeoapi_es  |     return f(get_current_context(), *args, **kwargs)
pygeoapi_es  |   File "/pygeoapi/pygeoapi/openapi.py", line 937, in generate
pygeoapi_es  |     content = generate_openapi_document(
pygeoapi_es  |   File "/pygeoapi/pygeoapi/openapi.py", line 884, in generate_openapi_document
pygeoapi_es  |     s = yaml_load(cfg_file)
pygeoapi_es  |   File "/pygeoapi/pygeoapi/util.py", line 199, in yaml_load
pygeoapi_es  |     return yaml.load(fh, Loader=EnvVarLoader)
pygeoapi_es  |   File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 114, in load
pygeoapi_es  |     return loader.get_single_data()
pygeoapi_es  |   File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 51, in get_single_data
pygeoapi_es  |     return self.construct_document(node)
pygeoapi_es  |   File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 60, in construct_document
pygeoapi_es  |     for dummy in generator:
pygeoapi_es  |   File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 413, in construct_yaml_map
pygeoapi_es  |     value = self.construct_mapping(node)
pygeoapi_es  |   File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 218, in construct_mapping
pygeoapi_es  |     return super().construct_mapping(node, deep=deep)
pygeoapi_es  |   File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 143, in construct_mapping
pygeoapi_es  |     value = self.construct_object(value_node, deep=deep)
pygeoapi_es  |   File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 100, in construct_object
pygeoapi_es  |     data = constructor(self, node)
pygeoapi_es  |   File "/pygeoapi/pygeoapi/util.py", line 185, in env_constructor
pygeoapi_es  |     raise EnvironmentError(
pygeoapi_es  | OSError: Could not find the 'PORT' environment variable
pygeoapi_es  | ERROR: openapi.yml could not be generated ERROR
pygeoapi_es  | ES failed: 255, exit
pygeoapi_es exited with code 1
```
It looks like the PORT is not exported from .env automatically without the explicit direction to export these variables to environment (and that direction is provided by env_file). So I don't think env_file can be NOT specified here, even if it is just `.env`.

[the remark this answers is not in the document]
Comment (doublebyte1) [reply docker/elastic/docker-compose.yml:38]: @KilimAnnejaro I think you may be using an old version of docker-compose. Can you type the output of:

`docker compose version`

[the remark this answers is not in the document]
Comment (doublebyte1) [reply docker/elastic/docker-compose.yml:38]: @KilimAnnejaro you are right in that it does not work without specifying the .env

**Pull request merged**